### ISSUE TITLE
Beets ignores replaygain backend

### ISSUE THREAD

**Issue by traverseda** [beetbox/beets#3471]
Beets ignores replaygain backend option entirely. I tested it in a fresh virtualenv.

### Problem

```sh
(test-replaygain)  traverseda@athame test-replaygain $ beet -vv replaygain                                                                                     
user configuration: /home/traverseda/.config/beets/config.yaml
data directory: /home/traverseda/.config/beets
plugin paths: 
Sending event: pluginload
library database: /home/traverseda/Music/beets.db
library directory: /home/traverseda/Music/tagged
Sending event: library_opened
replaygain: analyzing ミラクルミュージカル - Hawaii: Part II - Introduction to the Snow
error: replaygain initialization failed: Is bs1770gain installed? Is your method in config correct?
```

### Setup

* OS: Archlinux 
* Python version: 3.8.1
* beets version: 1.4.9
* Turning off plugins made problem go away (yes/no):  Turning off chroma did not help.

My configuration (output of `beet config`) is:

```yaml
directory: /home/traverseda/Music/tagged/
library: /home/traverseda/Music/beets.db
import:
  move: yes

replaygain:
    backend: gstreamer 

plugins: chroma replaygain
```

**Comment by sampsyo** [comment]
Hmm—that's odd. I can't reproduce this here:

```
$ beet -vv ls
user configuration: /Users/asampson/.config/beets/config.yaml
data directory: /Users/asampson/.config/beets
plugin paths: 
Sending event: pluginload
error: replaygain initialization failed: Failed to load GStreamer: python-gi not found
```

WIth this configuration:

```yaml
plugins: export replaygain
replaygain:
    backend: gstreamer
```

Are you sure you're using that configuration file? Can you try disabling the plugin and seeing whether that has any effect?

**Comment by traverseda** [comment]
It does indeed give the `replaygain initialization failed: Failed to load GStreamer: python-gi not found` before you run `pip install pygobject`. I think if you install pygobject you'll be able to reproduce? The same issue happens with all other replaygain backends.

It seems to be trying to initialize the plugin, but when it goes to actually use it it falls back to `bs1770gain`.

**Comment by traverseda** [comment]
This is fixed on `master/ca3142b1d92d177a2d789775a4ff2ee5569b9ba6`. Not sure when that fix was introduced though.

**Comment by traverseda** [comment]
Good news though, when I went  to dig into the problem it turns out it was already fixed. I'll just run master for now.
